package.json

```diff
@@ -42,28 +42,31 @@
     "@nuxtjs/proxy": "^1.2.4",
     "@nuxtjs/pwa": "2.0.8",
     "@nuxtjs/sitemap": "0.1.1",
+    "babel-polyfill": "^6.26.0",
     "dotenv": "^6.0.0",
     "dotenv-safe": "^6.0.0",
     "normalize.css": "^8.0.0",
-    "nuxt": "^1.0.0",
+    "nuxt": "2.1.0",
```

Comment:
Maybe update `nuxt` and other dependencies to the latest versions? Since the PR is about updating but is a little outdated